i18n/zh/docusaurus-plugin-content-docs/version-v1.5/reference/karmadactl/karmadactl-usage-conventions.md

```diff
@@ -202,6 +189,12 @@ karmadactl interpret -f resourceinterpretercustomization.yaml --observed-file ob
 karmadactl interpret -f resourceinterpretercustomization.yaml --desired-file desired-deploy-nginx.yaml --observed-file observed-deploy-nginx.yaml --operation Retain
 ```
 
+#### 执行 ReviseReplica 规则
```

Comment:
Do we need to apply this change in version-1.4?

Reply:
From one side, the documentation for karmada v1.4 is no longer actively maintained. 
From the other side, [V1.4.0](https://github.com/karmada-io/karmada/releases/tag/v1.4.0) was released on November 30, 2022, and [`pkg/karmadactl/interpret/interpret.go`](https://github.com/karmada-io/karmada/commits/master/pkg/karmadactl/interpret/interpret.go) had the full functionality of `karmadactl interpret` on November 28, 2022. Chronologically, the documentation can be updated.